WordPress/Classes/Models/Post.swift

```diff
@@ -204,6 +240,15 @@ class Post: AbstractPost {
 
                 return true
             }
+
+            if publicizeMessage ?? "" != originalPost.publicizeMessage ?? "" {
+                return true
+            }
+
+            if (!NSDictionary(dictionary: disabledPublicizeConnections ?? [:])
+                             .isEqual(to: originalPost.disabledPublicizeConnections ?? [:])) {
+                return true
```

Comment:
They can as long as their keys and values are equatable, but Dictionary is not equatable.